lib/helpers/getDataRouteForRoute.js

```diff
@@ -10,8 +10,7 @@ const buildId = fileContents.toString();
 // Return the data route for the given route
 const getDataRouteForRoute = (route) => {
   const filePath = getFilePathForRoute(route, "json");
-
-  return join("/_next", "data", buildId, filePath);
+  return `/_next/data/${buildId}${filePath}`;
```

Comment:
Paths in the `_redirects` file must use forward slashes. However, `path.join()` use backslashes on Windows. 
Note that `filePath` is not prepended with `/`, otherwise this would create a double slash (which was previously normalized thanks to `join()`.